Sources/VaporToolbox/New/TemplateScaffolder.swift

```diff
@@ -151,6 +152,22 @@ struct TemplateScaffolder {
     }
 }
 
+fileprivate extension StringProtocol {
+    func kebabed() -> String {
+        var output: String = self.first!.lowercased()
+        for char in self.dropFirst() {
+            if char.isWhitespace {
+                output.append("-")
+            } else if char.isUppercase {
+                output.append("-\(char.lowercased())")
+            } else {
+                output.append(char)
+            }
+        }
+        return output
+    }
```

Comment:
```suggestion
    func kebabed() -> String {
        return .init(self
            .flatMap { $0.isWhitespace ? "-" : "\($0)" }
            .flatMap { $0.isUppercase ? "-\($0.lowercased())" : "\($0)" }
        )
    }
```

Reply:
Okay, I committed the suggestion and then realized I didn't test it and there's an issue with it. If it is the first character is capitalized (the original issue) then this would happen:

```swift
"Name".kebabed() // -name
```

I'm not familiar enough with `flatMap` know how one would get around that